assets/js/app.js

```diff
@@ -129,8 +129,11 @@ window.addEventListener("lb:session_list:on_selection_change", () => {
     "#edit-sessions [name='disconnect']"
   );
   const closeAll = document.querySelector("#edit-sessions [name='close_all']");
+
   disconnect.disabled = !anySessionSelected;
+  disconnect.setAttribute("data-js-set-disabled", !anySessionSelected);
```

Comment:
We never use the attribute value, no?

```suggestion
  disconnect.setAttribute("data-js-keep-disabled", "");
```